akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1356,7 +1356,7 @@ trait FlowOps[+Out, +Mat] {
    * '''Cancels when''' downstream cancels
    */
   def throttle(elements: Int, per: FiniteDuration, maximumBurst: Int, mode: ThrottleMode): Repr[Out] =
-    throttle(elements, per, maximumBurst, _ ⇒ 1, mode)
+    throttle(elements, per, maximumBurst, ConstantFun.oneInt, mode)
```

Comment:
we should add in the ScalaDoc that the resulting rate will be lower than requested unless `maximumBurst.toDouble / elements * per > 2 * schedulerPeriod`